bazel/external/openssl_includes.BUILD

```diff
@@ -0,0 +1,18 @@
+cc_library(
```

Comment:
This is used to open SSL structure to allow access to alpn-related fields here: https://github.com/Maistra/envoy/pull/7/files#diff-399c9deb5d9027103b258cd84614c9b6R59

We continue to link against system ssl libraries, this is just to allow inclusion of internal openssl headers.